### ISSUE TITLE
Cannot compile tvision

### ISSUE THREAD

**Issue by HenningSvane** [magiblot/tvision#45]
I am running on Ubuntu 20.04 
Linux ubuntuclient 5.8.0-40-generic #45~20.04.1-Ubuntu SMP Fri Jan 15 11:35:04 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

I have downloaded the project and unzip it which created the folder structure
I the run:
hsv@ubuntuclient:~/tvision/tvision-master$ cmake . -B ./build -DCMAKE_BUILD_TYPE=Release && cmake --build ./build
-- Install path: /usr/local
-- Build Examples: ON
-- Build w/GPM: ON
-- gpm library requested
CMake Warning at CMakeLists.txt:36 (message):
  gpm library requested but not found
Call Stack (most recent call first):
  source/CMakeLists.txt:113 (tv_message)

CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
NCURSESW
    linked by target "tvision" in directory /home/hsv/tvision/tvision-master/source

-- Configuring incomplete, errors occurred!
See also "/home/hsv/tvision/tvision-master/build/CMakeFiles/CMakeOutput.log".
See also "/home/hsv/tvision/tvision-master/build/CMakeFiles/CMakeError.log".

[CMakeError.log](https://github.com/magiblot/tvision/files/5883491/CMakeError.log)
[CMakeOutput.log](https://github.com/magiblot/tvision/files/5883493/CMakeOutput.log)

What should I do different to get it compiled?
Regards
Henning

**Comment by magiblot** [comment]
Hi Henning! You are very close. You need to install the package `libncursesw5-dev`. I'm sorry the error message is not very helpful. I should probably list the required packages in the Readme.

Cheers.

**Comment by HenningSvane** [comment]
Hi
Thanks for a fast answer.
I was looking at that, and could see that libncursesw6 was installed. Not dev, but should it be version 5 and not the current version 6?

Regards
Henning Svane

________________________________
Fra: magiblot <notifications@github.com>
Sendt: torsdag den 28. januar 2021 01.21
Til: magiblot/tvision
Cc: HenningSvane; Author
Emne: Re: [magiblot/tvision] Cannot compile tvision (#45)


Hi Henning! You are very close. You need to install the package libncursesw5-dev. I'm sorry the error message is not very helpful. I should probably list the required packages in the Readme.

Cheers.

**Comment by magiblot** [comment]
That would seem logical, but there is no package `libncursesw6-dev`. Don't worry about that, it won't cause conflicts (I hope).

**Comment by magiblot** [comment]
After investigating a bit, I realized that `libncurses-dev` already provides the 'wide char' variant of Ncurses (not because Turbo Vision uses wide chars, but because the non-wide variant of Ncurses did was unable to display UTF-8 properly the last time I tried it). Actually, `libncursesw5-dev` consists of just a document ([file list](https://packages.ubuntu.com/groovy/amd64/libncursesw5-dev/filelist)) and has `libncurses-dev` as dependency. So whether you install `libncurses-dev` or `libncursesw5-dev`, the result will be the same.

Cheers.
